package/crossplane.yaml

```diff
@@ -2,3 +2,11 @@ apiVersion: meta.pkg.crossplane.io/v1
 kind: Provider
 metadata:
   name: provider-okta
+  annotations:
+    meta.crossplane.io/maintainer: <ADD EMAIL>
```

Comment:
@rjop-hccgt feel free to update the placeholders here in the package meta. These are all supported per the xpkg [specification](https://github.com/crossplane/crossplane/blob/master/contributing/specifications/xpkg.md?plain=1#L189) and will render into the listing on the Marketplace.

Reply:
done @jastang